pkg/apis/stork/v1alpha1/backuplocation.go

```diff
@@ -128,6 +128,8 @@ type GoogleConfig struct {
 
 type NfsConfig struct {
 	NfsServerAddr string `json:"nfsServerAddr"`
```

Comment:
Minor comment, it is better to change NfsServerAddr to ServerAddr :)

Reply:
I tried that but that is causing a circular dependencies... Like if I change that then in the utils file of kdmp expects in this file the variable to be Nfsbased
../../vendor/github.com/portworx/kdmp/pkg/drivers/utils/utils.go:362:73: backupLocation.Location.NfsConfig.ServerAddr undefined (type *"github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1".NfsConfig has no field or method ServerAddr)

Now I need to push this stork path forcefully if I chnage this variable and then vendor it kdmp to mitigatecompilation  issue in kdmp then vendor it back in stork  for stork to compile fine..So  someone has to merge with a broken patch..

Reply:
It can be done but there will be commit for which stork branch will not compile.. Which I thought i should aoid